lib/src/timetable.dart

```diff
@@ -52,13 +61,27 @@ class Timetable<E extends Event> extends StatelessWidget {
   /// out.
   final OnEventBackgroundTapCallback onEventBackgroundTap;
 
+  /// Builder for Week Indicator area.
+  ///
+  /// If it's not provided, or builder returns `null`,
+  /// default `WeekIndicator` widget will be used
+  final HeaderBuilder weekIndicatorBuilder;
+
+  /// Builder for Day header.
+  ///
+  /// If it's not provided, or builder returns `null`,
+  /// default `DateHeader` widget will be used
+  final HeaderBuilder dayHeaderBuilder;
```

Comment:
I try to differentiate between the terms day and date in this repo:
- **day:** day of the week
- **date:** date in the month, or just any date in general

(Also applies to `TimetableHeader`)
```suggestion
  final HeaderBuilder dateHeaderBuilder;
```

Reply:
done